index.html

```diff
@@ -5,9 +5,158 @@
+  <h1>SPARK TRAINING</h1>
+  <h1 class="date">2022</h1>
+</div>
+<div class="content">
+  <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Atque quisquam cum fugit nulla rerum aliquam quia commodi debitis obcaecati sequi ut vero, aut rem sint totam dolorum laborum accusantium suscipit! </p>
+</div>
+<h1>2022.05.27(FRI) ~ 28(SAT)</h1>
+<p>@ Kamzzy Creative Tech Hub Lagos State Nigeria</p>
+</section>
+  <section class="program">
+    <div class="checkDiv">
+    <input type="checkbox" class="toggle" id="toggle">
+    <label for="toggle">Main Program</label> 
+    <hr class="hr">
+  <div class="row gy-2 toggled">
+    <div class="col-md-2 main-col">
+      <div class="row">
+      <div class="imgDiv col-2 col-md-12">
+        <img src="/icons/training.png" alt="partnerLogoicons"/>
+      </div>
+      <div class="titleDiv col-3 col-md-12">
+        <h4>Training </h4>
+      </div>
+      <div class="contentDiv col-7 col-md-12">
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+      </div>
+      </div>
+</div>
+<div class="col-md-2 main-col">
+  <div class="row">
+      <div class="imgDiv col-2 col-md-12">
+        <img src="/icons/2d.png" alt="partnerLogoicons"/>
+      </div>
+      <div class="titleDiv col-3 col-md-12">
+        <h4>2D shapes</h4>
+      </div>
+      <div class="contentDiv col-7 col-md-12">
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+      </div>
+</div></div>
+<div class="col-md-2 main-col">
+  <div class="row">
+      <div class="imgDiv col-2 col-md-12">
+        <img src="/icons/3d.png" alt="partnerLogoicons"/>
+      </div>
+      <div class="titleDiv col-3 col-md-12">
+        <h4>3D shapes</h4>
+      </div>
+      <div class="contentDiv col-7 col-md-12">
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+      </div>
+      </div></div>
+      <div class="col-md-2 main-col">
+        <div class="row">
+      <div class="imgDiv col-2 col-md-12">
+        <img src="/icons/motion.png" alt="partnerLogoicons"/>
+      </div>
+      <div class="titleDiv col-3 col-md-12">
+        <h4>Add motion</h4>
+
+        </div>
+      <div class="contentDiv col-7 col-md-12">
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+      </div>
+      </div></div>
+      <div class="col-md-2 main-col">
+        <div class="row">
+      <div class="imgDiv col-2 col-md-12">
+        <img src="/icons/filter.png" alt="partnerLogoicons"/>
+      </div>
+      <div class="titleDiv col-3 col-md-12">
+        <h4>Add filter</h4>
+        </div>
+      <div class="contentDiv col-7 col-md-12">
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+      </div>
+      </div></div>
+    <div class="col-12 home-action-div">
+      <a class="home-action" href="#">Join AR Spark Training 2022 </a>
+    </div>
+  
+    <div class="speakers">
+      <h3>Featured Speakers</h3>
+      <hr class="hr">
+  </div>
+  </div>
+  </div>
```

Comment:
- [ ] It would be nice to show all your elements once you load the page instead of having to click the `Main program` to open them, this way we can improve the user experience as well. 
- [ ] Also, if you check the given template on Figma, you will see that the `Featured speakers` section has, at least, 6 people being shown on the screen, aligning them on two columns, so it would be nice if you do that instead of showing only 2 people and aligning them one behind the other. 
Here's the Figma template so you can see:
![Captura de pantalla 2022-04-21 003511](https://user-images.githubusercontent.com/59100477/164336039-0489fb15-6758-4e6c-8fdf-d42a064e8aaf.jpg)